### PR TITLE
fix(DB/Creature): fix spout emote text for The Lurker Below

### PR DESCRIPTION
**Pull request by zer0pwn**
## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [X] Database (SAI, creatures, etc).

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/19362

## Tests Performed:

This PR has been:
- [X] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1720992304054237637.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_text` SET `Text` = '%s takes a deep breath!' WHERE `creatureId` = 21217;
+UPDATE `broadcast_text` SET `MaleText` = '%s takes a deep breath!' WHERE `ID` = '20021';
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/rev_1720992304054237637.sql:3]:
```suggestion
UPDATE `creature_text` SET `Text` = '%s takes a deep breath!', `BroadcastTextId` = 0 WHERE `creatureId` = 21217;
```
Broadcast texts are not to be meddled with currently, as they've been bruteforced from Blizzard. It just so happens that they were taken from a Mists of Pandaria build where a bunch of texts (seems to include this one) were changed.

**Comment by avarishd** [comment]
BroadcastTextId is 20774
``` sql 
SELECT * FROM broadcast_text WHERE maletext LIKE "%%s takes a deep breath!%";
```

**Comment by zer0pwn** [comment]
@avarishd I think that might be the BroadcastTextId for Grauf, just similar text

```sql
SELECT * FROM creature_text WHERE BroadcastTextId = '20774'
```

image

**Comment by heyitsbench** [comment]
> avarishd I think that might be the BroadcastTextId for Grauf, just similar text

Grauf emote
I think he might be using the wrong broadcast text as well then.

**Review by Gultask — APPROVED** [review]
I just added the BroadcastID, some ANDs in the query, nothing too serious :)

**Comment by Gultask** [comment]
Oh! Thanks and congrats on the first PR!

**Pull request merged**